README.md

```diff
@@ -1,5 +1,5 @@
 
-[![Build Status](https://travis-ci.org/flexion/hud-disaster-data.svg?branch=master)](https://travis-ci.org/flexion/hud-disaster-data)
+Staging [![Build Status](https://travis-ci.org/flexion/hud-disaster-data.svg?branch=master)](https://travis-ci.org/flexion/hud-disaster-data)
```

Comment:
The effect of this is that the status of the _staging_ repo is displayed on the master repo, right? May I ask that we work toward a solution, so that the build status of _this_ repo is displayed here, instead?